contrib/grafana/synapse.json

```diff
@@ -7,14 +7,22 @@
       "type": "datasource",
       "pluginId": "prometheus",
       "pluginName": "Prometheus"
+    },
+    {
+      "name": "DS_JKI.RE",
+      "label": "jki.re",
+      "description": "",
+      "type": "datasource",
+      "pluginId": "prometheus",
+      "pluginName": "Prometheus"
```

Comment:
You probably don't want to add this bit in :slightly_smiling_face:

Reply:
Woops, thanks!

Reply:
(this was because one of the graphs had accidentally been set to always be for `jki.re`)